samples/net/sockets/can/src/main.c

```diff
@@ -60,7 +60,7 @@ static void tx(int *can_fd)
 	LOG_DBG("Sending CAN data...");
 
 	while (1) {
-		ret = send(fd, &frame, sizeof(frame), K_FOREVER);
+		ret = send(fd, &frame, sizeof(frame), SYS_FOREVER_MS);
```

Comment:
Will this not pass SYS_FOREVER_MS as the flags for the socket send() function?

Reply:
Yes. @jukkar what does the socket expect here?

Reply:
The SYS_FOREVER_MS which has a value -1, is currently ignored as the value is expected to be a flag, and the -1 is not a proper flag value. The default in send() is to wait forever so the end result is the same. If one does not want to wait at all, then MSG_DONTWAIT value should be supplied here or socket can be made non blocking.
Best here, would be to set the flags to value 0 in order not to confuse the reader.

Reply:
Done